### DISCUSSION TITLE
add max_digits support to `conint`, simliar to `condecimal`

### DISCUSSION THREAD

**Discussion by Ricyteach** [pydantic/pydantic#3831] [Feature Request]
Hello: I have just started using pydantic and it's great. Thanks for the hard work.

I am working on a project for which it would be helpful in some cases to add max_digits argument support to `conint`, similar to `condecimal`. Is this possible? Or should I look into making a [custom validation type](https://pydantic-docs.helpmanual.io/usage/types/#custom-data-types)? The custom validation type is certainly a possibility, it's just a bit scary to me as a new user and it seems like it would be a simple feature to add to `conint`.

EDIT: the reason I want a max_digits feature is I am using the max_digits as meta-data for field width in an application.

I have fields that contain str, decimal, or int values that might be, for example, 5 characters wide:

```
<space><space><space><space>1
abc<space><space>
<space><space>1.0
```

Using pydantic I am telling my application to look at `constr.max_length` and `condecimal.max_digits` to determine the field widths. But I can't use `conint.le` or `conint.lt` to determine the field widths for some integer values, because the field size in some cases is bigger than the field width of the maximum integer.

An example might be:

```
x=conint(max_digits=3, le=10)
```

In this case pydantic would validate against the le value but I would still be able to find the field width using `x.max_digits`.

**Comment by MuriloCarlos1567** [comment]
@Ricyteach I already needed the same feature. I implemented in my project a function to replace the conint().
Can I work on this implementation?
